lisa/features/__init__.py

```diff
@@ -15,6 +16,7 @@
 from .startstop import StartStop
 
 __all__ = [
+    "DataDisk",
```

Comment:
The data disk doesn't need to be a feature. It's values should be translated in the Azure platform. As we added attributes, they are not part of feature, they are similar like nic_count, core_count.